src/core.constants.js

```diff
@@ -4,7 +4,7 @@ const SHOW_CLASS = 'octotree-show';
 const PINNED_CLASS = 'octotree-pinned';
 
 const STORE = {
-  TOKEN: 'octotree.access_token',
```

Comment:
This breaks saved token on user, is this really necessary for changing the name ?

Reply:
I discussed with @buunguyen and came up with this naming convention. I think it worths it